pandas/io/orc.py

```diff
@@ -44,8 +44,8 @@ def read_orc(
     path: FilePath | ReadBuffer[bytes],
     columns: list[str] | None = None,
     dtype_backend: DtypeBackend | lib.NoDefault = lib.no_default,
-    filesystem=None,
-    **kwargs,
+    filesystem: Any = None,
```

Comment:
fsspec/pyarrow are installed on the CI, could add the appropriate types if possible